libpod/runtime_ctr.go

```diff
@@ -915,6 +915,44 @@ func (r *Runtime) evictContainer(ctx context.Context, idOrName string, removeVol
 	return id, cleanupErr
 }
 
+// DependMap populates of map of all containers depending on the specified container
+func (r *Runtime) DependMap(ctx context.Context, ctr *Container, ctrMap map[string]*Container) error {
```

Comment:
Maybe change the signature to not accept a `map[string]*Container` but to allocate it here and return it instead? I don't think many people will call DependMap on partially-generated maps?

Reply:
I am passing in the map with the list of all the containers currently added in the podman run command.  This prevents us from having to lookup containers that are currently being deleted.